internal/adapters/cloudformation/aws/rds/instance.go

```diff
@@ -58,3 +71,62 @@ func getClustersAndInstances(ctx parser.FileContext) (clusters []rds.Cluster, or
 
 	return clusters, orphans
 }
+
+func getDBParameterGroups(r *parser.Resource) (dbpmgroup []rds.DBParameterGroupsList) {
+
+	DBParameterGroup := r.GetProperty("DBParameterGroups")
+
+	if DBParameterGroup.IsNil() || DBParameterGroup.IsNotNil() {
+		return dbpmgroup
+	}
```

Comment:
I don't understand this logic. Wouldn't this be always true?

Reply:
that is a mistake. I have resolved this.